app/grandchallenge/algorithms/models.py

```diff
@@ -293,9 +294,11 @@ def is_editor(self, user):
         return user.groups.filter(pk=self.editors_group.pk).exists()
 
     def add_editor(self, user):
+        follow(user=user, obj=self, actor_only=False, send_action=False)
         return user.groups.add(self.editors_group)
 
     def remove_editor(self, user):
+        unfollow(user=user, obj=self, send_action=False)
```

Comment:
I don't think adding the events here is the best way to go as there are many ways that an editor can be added and removed from the group and we cannot force all codepaths down this route. I think what is needed is to add a signal handler on `m2m_changed` of the groups members, those are quite tricky to set up but there are a few examples in the codebase, let me know and I can point you to them.